lib/notification-support.ts

```diff
@@ -27,11 +27,20 @@ function supportsDarwinNotifications() {
 }
 
 function supportsWindowsNotifications() {
+  // The Windows Notification APIs we use are only available on Windows 10+, and
+  // some of them are limited before the Creators Update (build 15063).
+  // See: https://docs.microsoft.com/en-us/uwp/api/windows.ui.notifications.toastnotification.tag?view=winrt-22621#remarks
+  const CreatorsUpdateBuildNumber = 15063
+
   const versionComponents = os.release().split('.')
   const majorVersion = parseInt(versionComponents[0], 10)
+  const buildNumber =
+    versionComponents.length >= 3
+      ? parseInt(versionComponents[2], 10)
+      : CreatorsUpdateBuildNumber
 
-  // Only Windows 10 and newer are supported
-  return majorVersion >= 10
+  // Only Windows 10 (15063) and newer are supported
+  return majorVersion >= 10 && buildNumber >= CreatorsUpdateBuildNumber
```

Comment:
Actually this is wrong… 🤦  it should be:
```suggestion
  return majorVersion > 10 || (majorVersion === 10 && buildNumber >= CreatorsUpdateBuildNumber)
```